packages/cli/README.md

```diff
@@ -101,6 +101,28 @@ To transform your project stylesheets to target JavaScript modules containing th
 $ stc --srcDir="./src" --outDir="./dist"
 ```
 
+## Usage `stc-format`
+
+After installing `@stylable/cli`, a new `stc-format` command will be available, running `stc-format --help` will provide a brief description for the options available.
```

Comment:
"a new"? sounds like release notes.
also `--help` is pretty standard and is just below in the option list, I don't you need to mention it above

Reply:
Removing the new statement. 
The help statement is the same as listed above for `stc`, I see no harm in including it.